controllers/helpers_test.go

```diff
@@ -84,3 +90,154 @@ func getClusterScope(awsCluster infrav1.AWSCluster) (*scope.ClusterScope, error)
 		},
 	)
 }
+
+func mockedCreateLBCalls(m *mock_elbiface.MockELBAPIMockRecorder) {
+	m.DescribeLoadBalancers(gomock.Eq(&elb.DescribeLoadBalancersInput{
+		LoadBalancerNames: aws.StringSlice([]string{"test-cluster-apiserver"}),
+	})).
+		Return(&elb.DescribeLoadBalancersOutput{
+			LoadBalancerDescriptions: []*elb.LoadBalancerDescription{
+				{
+					Scheme:            aws.String(string(infrav1.ClassicELBSchemeInternetFacing)),
+					Subnets:           []*string{aws.String("subnet-1")},
+					AvailabilityZones: []*string{aws.String("us-east-1a")},
+					VPCId:             aws.String("vpc-exists"),
+				},
+			},
+		}, nil).MinTimes(1)
+	m.DescribeLoadBalancerAttributes(gomock.Eq(&elb.DescribeLoadBalancerAttributesInput{
+		LoadBalancerName: aws.String("test-cluster-apiserver"),
+	})).
+		Return(&elb.DescribeLoadBalancerAttributesOutput{
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				CrossZoneLoadBalancing: &elb.CrossZoneLoadBalancing{
+					Enabled: aws.Bool(false),
+				},
+			},
+		}, nil)
+	m.DescribeTags(&elb.DescribeTagsInput{LoadBalancerNames: []*string{aws.String("test-cluster-apiserver")}}).Return(
+		&elb.DescribeTagsOutput{
+			TagDescriptions: []*elb.TagDescription{
+				{
+					LoadBalancerName: aws.String("test-cluster-apiserver"),
+					Tags: []*elb.Tag{{
+						Key:   aws.String(infrav1.ClusterTagKey("test-cluster-apiserver")),
+						Value: aws.String(string(infrav1.ResourceLifecycleOwned)),
+					}},
+				},
+			},
+		}, nil)
+	m.ModifyLoadBalancerAttributes(gomock.Eq(&elb.ModifyLoadBalancerAttributesInput{
+		LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+			ConnectionSettings:     &elb.ConnectionSettings{IdleTimeout: aws.Int64(600)},
+			CrossZoneLoadBalancing: &elb.CrossZoneLoadBalancing{Enabled: aws.Bool(false)},
+		},
+		LoadBalancerName: aws.String(""),
+	})).MaxTimes(1)
+	m.AddTags(gomock.Eq(&elb.AddTagsInput{
```

Comment:
In our code, `AddTags` return output is not checked and we are not using any error scenarios where `AddTags` will return error, so that's why not added any return behavior